infrastructure/main.tf

```diff
@@ -93,6 +93,9 @@ module "bulk-scan" {
     STORAGE_ACCOUNT_NAME = "${local.storage_account_name}"
     STORAGE_KEY          = "${local.storage_account_primary_key}"
     STORAGE_URL          = "${local.storage_account_url}"
+    PROXY_ENABLED        = "${var.proxy_enabled}"
```

Comment:
Maybe this should have the same prefix as the other two? Up to you.

Reply:
I'll rename it